mava/systems/ppo/anakin/ff_ippo.py

```diff
@@ -253,39 +232,36 @@ def _critic_loss_fn(
                     (critic_grads, critic_loss_info), axis_name="device"
                 )
 
-                # UPDATE ACTOR PARAMS AND OPTIMISER STATE
+                # Update params and optimiser state
                 actor_updates, actor_new_opt_state = actor_update_fn(
                     actor_grads, opt_states.actor_opt_state
                 )
                 actor_new_params = optax.apply_updates(params.actor_params, actor_updates)
 
-                # UPDATE CRITIC PARAMS AND OPTIMISER STATE
                 critic_updates, critic_new_opt_state = critic_update_fn(
                     critic_grads, opt_states.critic_opt_state
                 )
                 critic_new_params = optax.apply_updates(params.critic_params, critic_updates)
 
-                # PACK NEW PARAMS AND OPTIMISER STATE
                 new_params = Params(actor_new_params, critic_new_params)
                 new_opt_state = OptStates(actor_new_opt_state, critic_new_opt_state)
 
-                # PACK LOSS INFO
-                total_loss = actor_loss_info[0] + critic_loss_info[0]
-                value_loss = critic_loss_info[1]
-                actor_loss = actor_loss_info[1][0]
-                entropy = actor_loss_info[1][1]
+                actor_loss, (raw_actor_loss, entropy) = actor_loss_info
+                critic_loss, raw_critic_loss = critic_loss_info
+
+                total_loss = actor_loss + critic_loss
                 loss_info = {
                     "total_loss": total_loss,
-                    "value_loss": value_loss,
-                    "actor_loss": actor_loss,
+                    "value_loss": raw_critic_loss,
```

Comment:
If we change value -> critic I think we should be consistent with the naming everywhere. Are you keeping it as value to still be comparable with old experiments?